### ISSUE TITLE
Download archive page subtitle plugin support

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/themedd#163]
This is merely just an idea and it might be better placed as a snippet or child theme. With that said I feel it _could_ conditionally work with Themedd core as well, hence opening this discussion.

I use the popular _Subtitles_ plugin on a lot of my WP sites and it integrates perfectly with Themedd, however since the download archive now has title support, It would be useful if the subtitles plugin was installed to be able to set the subtitle as well as the title from within the customizer. This would help keep congruency across all the pages (categories, tags, pages, posts, etc.).

**Comment by davidsherlock** [comment]
For sake of clarity, this plugin: https://wordpress.org/plugins/subtitles/

**Comment by amdrew** [comment]
@davidsherlock To clarify, there would be a new input field shown underneath the new one to set the subtitle. This input field would only be shown if subtitles is installed and activated, and the EDD archive page hasn't been disabled. Is that correct?

This feels like a "nice to have", probably not something I see as being included at this point, unless it is requested a lot. For now, I think it is better as a code snippet or included from the child theme.

**Comment by davidsherlock** [comment]
That is correct @amdrew, and I do in part agree this isn’t “essential” per say but certainly useful to have to maintain uniformity across the site.
